modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -50,21 +50,29 @@
 
 /datum/reagent/medicine/nanite_slurry
 	name = "Nanite Slurry"
-	description = "If used in touch-based applications, immediately repairs and refurbishes synthetic lifeforms, also does that while circulating in their system."
+	description = "A localized swarm of nanomachines specialized in repairing mechanical parts. Due to the nanites needing to interface with the host's systems to repair them, a surplus of them will cause them to overheat, or for the swarm to forcefully eject out of the mouth of organics for safety."
 	reagent_state = LIQUID
 	color = "#cccccc"
-	process_flags = REAGENT_SYNTHETIC
+	overdose_threshold = 20
+	metabolization_rate = 1.25 * REAGENTS_METABOLISM
+	process_flags = REAGENT_SYNTHETIC | REAGENT_ORGANIC
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	/// How much brute and burn individually is healed per tick
+	var/healing = 1
+	/// How much body temperature is adjusted by per tick
+	var/temperature_change = 50
+
 
-/datum/reagent/medicine/nanite_slurry/expose_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1)
-	if(iscarbon(M))
-		if(!(method in list(INGEST, VAPOR, INJECT)))
-			M.adjustFireLoss(-reac_volume)
-			M.adjustBruteLoss(-reac_volume)
-			if(show_message)
-				to_chat(M, span_notice("You feel much better..."))
+/datum/reagent/medicine/nanite_slurry/on_mob_life(mob/living/carbon/affected_mob)
+	affected_mob.heal_bodypart_damage(healing*REM, healing*REM, required_status = BODYPART_ROBOTIC)
 	..()
 
-/datum/reagent/medicine/nanite_slurry/on_mob_life(mob/living/carbon/M)
-	M.heal_bodypart_damage(0.5*REM, 0.5*REM, required_status = BODYPART_ROBOTIC)
+/datum/reagent/medicine/nanite_slurry/overdose_process(mob/living/carbon/affected_mob, delta_time, times_fired)
+	if(affected_mob.mob_biotypes & MOB_ROBOTIC)
+		affected_mob.adjust_bodytemperature(temperature_change * REM * delta_time)
+		return ..()
+	affected_mob.reagents.remove_reagent(type, 3.6) //gets removed from organics very fast
+	if(prob(25))
```

Comment:
This also should be a define, like `#define NANITE_SLURRY_ORGANIC_VOMIT_CHANCE 25`